cli/src/cluster_query.rs

```diff
@@ -276,6 +295,55 @@ pub fn process_ping(
     Ok("".to_string())
 }
 
+pub fn process_show_validators(rpc_client: &RpcClient, use_lamports_unit: bool) -> ProcessResult {
+    let vote_accounts = rpc_client.get_vote_accounts()?;
+    let total_activate_stake = vote_accounts
+        .current
+        .iter()
+        .chain(vote_accounts.delinquent.iter())
+        .fold(0, |acc, vote_account| acc + vote_account.activated_stake)
+        as f64;
+
+    println!(
+        "{:<44}  {:<44}  {:>11}  {:>10}  {:>11}  {:>10}",
+        "Identity Pubkey", "Vote Pubkey", "Commission", "Last Vote", "Root Block", "Active Stake",
+    );
+
+    for vote_account in vote_accounts
+        .current
+        .iter()
+        .chain(vote_accounts.delinquent.iter())
+    {
+        fn non_zero_or_dash(v: u64) -> String {
+            if v == 0 {
+                "-".into()
+            } else {
+                format!("{}", v)
+            }
+        }
+
+        println!(
+            "{:<44}  {:<44}  {:>10.1}%  {:>10}  {:>11}  {}",
+            vote_account.node_pubkey,
+            vote_account.vote_pubkey,
+            f64::from(vote_account.commission) * 100.0 / f64::from(std::u8::MAX),
```

Comment:
What do you think about matching the activated stake format? `Integer (percentage%)`
That's what we're doing on the Network Explorer:
![image](https://user-images.githubusercontent.com/3030561/66792857-e468a580-eeb7-11e9-9e9d-a4e70606ab3c.png)

I tend to think the integer value is useful, since that's the unit validators need to use in the `--commission` argument

Reply:
I like it, added